src/core/include/mp-units/quantity.h

```diff
@@ -189,6 +191,25 @@ class quantity {
     return (*this).force_in(U{}).numerical_value_;
   }
 
+  // conversion operators
+  template<QuantityLike Q>
+  [[nodiscard]] explicit(is_specialization_of<decltype(quantity_like_traits<Q>::from_numerical_value(numerical_value_)),
+                                              convert_explicitly>) constexpr
+  operator Q() const& noexcept(noexcept(quantity_like_traits<Q>::from_numerical_value(numerical_value_)) &&
+                               std::is_nothrow_copy_constructible_v<rep>)
+  {
+    return quantity_like_traits<Q>::from_numerical_value(numerical_value_).value;
+  }
+
+  template<QuantityLike Q>
+  [[nodiscard]] explicit(is_specialization_of<decltype(quantity_like_traits<Q>::from_numerical_value(numerical_value_)),
+                                              convert_explicitly>) constexpr
+  operator Q() const&& noexcept(noexcept(quantity_like_traits<Q>::from_numerical_value(numerical_value_)) &&
+                                std::is_nothrow_copy_constructible_v<rep>)
```

Comment:
Either apply this suggestion, or
remove this overload (the other one has equivalent effects).
```suggestion
  operator Q() && noexcept(noexcept(quantity_like_traits<Q>::from_numerical_value(numerical_value_)) &&
                                std::is_nothrow_move_constructible_v<rep>)
```